### PR TITLE
Final - Thuan Nguyen - DetailUI - Personal Acount View

### PR DESCRIPTION
**Pull request by mvn-thuannguyen-dn**

### CODE REVIEW

File: FinalProject/Views/Detail/Profile/ProfileViewController.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
+//
+//  ProfileViewController.swift
+//  FinalProject
+//
+//  Created by thuan.nguyen on 16/06/2022.
+//
+
+import UIKit
+
+class ProfileViewController: UIViewController {
```
Comment (mvn-annguyen-dn) [inline FinalProject/Views/Detail/Profile/ProfileViewController.swift:10]: final

File: FinalProject/Views/Detail/Profile/ProfileViewController.swift [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    // MARK: - Outlets
+    @IBOutlet private weak var tableView: UITableView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+    // MARK: - private funtions
```
Comment (mvn-annguyen-dn) [inline FinalProject/Views/Detail/Profile/ProfileViewController.swift:17]: line spacing. `function`

File: FinalProject/Views/Detail/Profile/ProfileViewController.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    private func setupNavigation() {
+        title = "Tài khoản cá nhân"
+        let settingButton = UIBarButtonItem(image: UIImage(imageLiteralResourceName: "ic-settings"), style: .plain, target: self, action: #selector(settingsAction))
+        navigationItem.rightBarButtonItem = settingButton
+    }
+    private func configTableView() {
```
Comment (mvn-annguyen-dn) [inline FinalProject/Views/Detail/Profile/ProfileViewController.swift:23]: line space

File: FinalProject/Views/Detail/Profile/ProfileViewController.swift [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "cell")
+    }
+    private func setupUI() {
```
Comment (mvn-annguyen-dn) [inline FinalProject/Views/Detail/Profile/ProfileViewController.swift:28]: line space

File: FinalProject/Views/Detail/Profile/ProfileViewController.swift [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        setupNavigation()
+        configTableView()
+    }
+    // MARK: - Actions
+    @objc private func settingsAction() {
```
Comment (mvn-annguyen-dn) [inline FinalProject/Views/Detail/Profile/ProfileViewController.swift:33]: line space

File: FinalProject/Views/Detail/Profile/ProfileViewController.swift [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+    }
+}
+// MARK: - UITableView
+extension ProfileViewController: UITableViewDelegate, UITableViewDataSource {
```
Comment (mvn-annguyen-dn) [inline FinalProject/Views/Detail/Profile/ProfileViewController.swift:38]: line space
separate `UITableViewDelegate` and `UITableViewDatasource`

**Review by mvn-annguyen-dn — CHANGES_REQUESTED** [review]
Please fix my comments and give a screenshot of UI Persional Account

**Review by mvn-annguyen-dn — COMMENTED** [review]
@mvn-thuannguyen-dn please upload screenshot UI

File: FinalProject/Views/Base/BaseTabBarController.swift [hunk 7]
```diff
@@ -16,7 +16,7 @@ final class BaseTabBarController: UITabBarController {
 
     // MARK: - Setup View Controller
     private func setupViewController() {
-        let homeViewController = HomeViewController()
+        let homeViewController = LiveViewController()
```
[the remark this answers is not in the document]
Comment (mvn-thuannguyen-dn) [reply FinalProject/Views/Base/BaseTabBarController.swift:19]: Vì HomeViewController em chưa merge vào tabbar nên em chỉ để tạm thời khỏi lỗi

**Comment by mvn-annguyen-dn** [comment]
Xin 500đ UI

**Review by mvn-annguyen-dn — COMMENTED** [review]
Check my comments

**Comment by mvn-thuannguyen-dn** [comment]
1. UI_ProfileViewController
Screenshot 2022-07-05 at 17 35 02
2. UI_PlayerViewController
Screenshot 2022-07-06 at 08 52 32

**Review by mvn-annguyen-dn — APPROVED** [review]

**Pull request merged**